js/djtilesource.js

```diff
@@ -1,92 +1,134 @@
+/*jslint browser: true*/
+/*global OpenSeadragon, Drupal*/
+/**
+ * @file
+ * Defines the source for exposing Djatoka to OpenSeadragon.
+ */
 (function($) {
+  'use strict';
+
+  // Remove IIFTileSource, as it assumes Djatoka responses belong to it for some reason.
+  $.IIIFTileSource = undefined;
```

Comment:
@whikloj: It's been a while, but I think I was just trying to track down _why_ `IIIFTileSource` assumes it should take control (and so has to be suppressed/set to `undefined`; the unspecified "some reason").